src/Toolkit/Str.php

```diff
@@ -973,16 +973,25 @@ public static function substr(string $string = null, int $start = 0, int $length
      *                                    - start: start placeholder
      *                                    - end: end placeholder
      *                                    A simple fallback string is supported for compatibility (but deprecated).
-     * @param string $start Placeholder start characters
-     * @param string $end Placeholder end characters
+     * @param string $start Placeholder start characters (deprecated)
+     * @param string $end Placeholder end characters (deprecated)
      *
-     * @todo Deprecate `string $fallback` and `$start`/`$end` arguments with warning in 3.6.0
      * @todo Remove `$start` and `$end` parameters, rename `$fallback` to `$options` and only support `array` type for `$options` in 3.7.0
      *
      * @return string The filled-in string
      */
     public static function template(string $string = null, array $data = [], $fallback = null, string $start = '{{', string $end = '}}'): string
     {
+        // @codeCoverageIgnoreStart
+        if (
+            is_string($fallback) === true ||
+            $start !== '{{' ||
+            $end !== '}}'
+        ) {
+            deprecated('Str::template(): The $fallback, $start and $end parameters have been deprecated. Please pass an araay to the $options parameter instead with `fallback`, `start` or `end` keys: Str::template($string, $data, $options)');
```

Comment:
I guess text should be improved:

- `araay` typo
- we don't have `$options` parameter yet

Reply:
Fixed the typo. On `$options`, I think we need to call it that already. In 3.7 we will officially rename it in the code, but we are treating it already like that. And would be weird to read "$fallback parameter has been deprecated, please use $fallback instead"

Reply:
Not important. I just thought it would be confusing if the developers couldn't find the `$options` parameter.